downstream/modules/platform/proc-aap-activate-with-credentials.adoc

```diff
@@ -5,6 +5,16 @@
 
 When {PlatformNameShort} launches for the first time, the {PlatformNameShort} Subscription screen automatically displays. You can use your Red Hat credentials to retrieve and import your subscription directly into {PlatformNameShort}.
 
+[Note]
+====
+You are opted in for {Analytics} by default when you activate the platform on first time log in. This helps Red Hat improve the product by delivering you a much better user experience. You can opt out, after activating {PlatformNameShort}, by doing the following: 
+
+. From the navigation panel, select menu:[Settings] and select the *Miscellaneous System settings* option.
+. Click btn:[Edit].
+. Switch the *Gather data for Automation Analytics* toggle off.
```

Comment:
Per the IBM style guide, avoid using terms 'toggle off/on'. In this instance, suggest using "Toggle the switch to the off position" or something similar.